2_process/src/create_GFv1_NHDv2_xwalk.R

```diff
@@ -30,6 +30,13 @@ create_GFv1_NHDv2_xwalk <- function(prms_lines, nhd_lines, prms_hrus,
     nhd_lines_select <- nhd_lines
   }
   
+  # Add special handling for split segments so that all split segments have a value
+  # for segidnat (instead of NA)
+  prms_lines_complete <- prms_lines %>%
+    group_by(subsegseg) %>%
+    mutate(segidnat = segidnat[!is.na(segidnat)]) %>%
+    ungroup()
```

Comment:
Yes, it does because we've assumed that the `segidnat` is the same for the split segments (e.g. 3_1 _and_ 3_2), but in `p1_reaches_sf` the _1 splits have `NA` for `segidnat` whereas the _2 splits don't. The manual assignment might be more readily interpretable/readable so long as those assignments won't change, but it is manually coding the `segidnat` for those rows.

Reply:
Okay, I didn't realize that the same segment is assigned to _1 and _2. That makes sense, thanks.